src/main/scala/no/ndla/articleapi/ArticleApiProperties.scala

```diff
@@ -35,7 +35,7 @@ object ArticleApiProperties extends LazyLogging {
   val MetaInitialConnections = 3
   val MetaMaxConnections = 20
 
-  val AttachmentStorageName = s"$Environment.attachments.ndla"
+  val AttachmentStorageName = if (Environment == "local") s"test.article-attachments.ndla" else s"$Environment.article-attachments.ndla"
```

Comment:
Mnjejoaa, det kan jo hende dersom man gjør noe rart lokalt. Ryddigere å bare lage en egen bøtte for lokalt, så slipper vi denne if sjekken